thrust/system/cuda/detail/bulk/detail/cuda_launcher/triple_chevron_launcher.hpp

```diff
@@ -71,11 +71,25 @@ struct triple_chevron_launcher_base<block_size,Function,true>
 {
   typedef void (*global_function_pointer_t)(Function);
 
+#if THRUST_DEVICE_COMPILER == THRUST_DEVICE_COMPILER_CLANG
+  __host__
+  static global_function_pointer_t global_function_pointer()
+  {
+    return launch_by_value<block_size,Function>;
+  }
+
+  __device__
+  static global_function_pointer_t global_function_pointer()
+  {
```

Comment:
Can we add a comment saying that clang doesn't support dynamic parallelism, and add assert(0) or something like this to terminate execution should this code path be ever taken at runtime?